aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/MSKFirehoseResponse.java

```diff
@@ -0,0 +1,54 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+
+import lombok.*;
+
+/**
+ * Response model for Amazon Data Firehose Lambda transformation with MSK as a source.
+ */
+
+@Data
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class MSKFirehoseResponse {
+
+    public enum Result {
+
+        /**
```

Comment:
I would put this information in the class javadocs, so that IDEs will display it, and make it easier to find.

Reply:
added information to class Javadocs.